### PR TITLE
Stringify to markdown

### PR DESCRIPTION
**Pull request by arobase-che**
Explicit enough amha.

I just want to talk about 1 thing. I think that with the test : 

```markdown
+----- A -----+
|             |
+-------------+
|             |
+-------------+
```

Is a regression since now it's parsed as a table. See the [snapshot](https://github.com/zestedesavoir/zmarkdown/blob/d446425880f9337c8eddadc0e3fdb0b7c296dca6/packages/remark-grid-tables/__tests__/__snapshots__/index.js.snap#L47).

With love 💌

### CODE REVIEW

File: packages/remark-grid-tables/src/index.js [hunk 1]
```diff
@@ -448,6 +448,228 @@ function transformer (tree) {
...
+
+    // "End" line
+    // It's the last line of the cell. Actually the border.
+    line = row[0].b ? '+' : '|'
+
+    row.forEach((cell, j) => {
+      let c = ' '
+
+      if (cell.b) {
+        if (row[0].H) {
```
Comment (artragis) [inline packages/remark-grid-tables/src/index.js]: can you use more "obvious" name such as `isHeader`? idem for cell.b and cell.r etc.

Comment (arobase-che) [reply packages/remark-grid-tables/src/index.js]: Rename it to `isHeader`, `hasBottom` and `hasRigth` ? 

I'm out of coke/chocolate now, so yes but i'm out for an hour. How ? As a new commit or an amend one ?

Thanks for the review. It's really appreciated.

Comment (artragis) [reply packages/remark-grid-tables/src/index.js]: I think you should fixup the new commit to the old one.

**Comment by AmarOk1412** [comment]
Ok (and hi btw, it's been a long time :( ). Here the answer (better than a PM)

# Empty grid

The min width of a column is 3. So, to get an empty table, you will write:

```
+---+
|   |
+---+
```

Not:
```
+--+
|  |
+--+
```

This is mainly for 2 reasons :

1. Empty table are not the main thing you want, and if there is a letter in a case, you will need at least 3 characters in the cell. (`| x |`)
2. Classic tables also needs at least 3 characters, so it's homogeneous.

# regression?

```
+----- A -----+
|             |
+-------------+
|             |
+-------------+
```

Is not a regression, but the behavior has changed yes. The thing is, now, we don't have to let a space between the table and the rest of the text.

```
This is an empty table:
+-------------+
|             |
+-------------+
```

Will give *This is an empty table:* followed by an empty table.

So,
```
+----- A -----+
|             |
+-------------+
|             |
+-------------+
```
will give 2 ugly lines followed by an empty table.

I think, this is logic, because we want this example as a valid grid-table followed by a text:

```
+---------+
| A       |
+---------+
Text at the end
```

So, this is the current behavior and I think it's the more logical thing to do, but if there is a consensus to change this, it's ok for me.

**Comment by arobase-che** [comment]
Hi @AmarOk1412 \o :tada: 

About empty grid, now, i set the min width to 3.  (last commit)

About the weird behavior. For this PR, it doesn't matter and i don't know witch one is better.

**Comment by AmarOk1412** [comment]
Indeed, nothing to do with this PR, just answering to your first post :)

**Comment by artragis** [comment]
@vhf do you agree to merge this one?

**Pull request merged**